scripts/bimg.lua

```diff
@@ -31,6 +33,8 @@ project "bimg"
 		path.join(ASTC_CODEC_DIR, "src/decoder/physical_astc_block.*"),
 		path.join(ASTC_CODEC_DIR, "src/decoder/quantization.*"),
 		path.join(ASTC_CODEC_DIR, "src/decoder/weight_infill.*"),
+
+		path.join(MINIZ_DIR, "miniz.*"),
```

Comment:
Just do the same thing you did in other files:

`path.join(BIMG_DIR, "3rdparty/tinyexr/deps/miniz"),` without `MINIZ_DIR`. It's part of TinyEXR.

Reply:
Ok.
Note that miniz is not a single header library anymore.

Reply:
Does require to be compiled as .lib or just including bunch of files?

Reply:
It can be built as a static library if needed. It's a single C file to add.